print/components/config/Picasso.vue

```diff
@@ -35,6 +38,8 @@ export default {
     this.periods = this.options.periods.map((periodUri) => {
       return this.$api.get(periodUri) // TODO prevent specifying arbitrary absolute URLs that the print container should fetch...
     })
+
+    this.landscape = this.options.orientation === 'L'
```

Comment:
Why is this a data property populated in `fetch`? Why not simply a computed property?

Reply:
Depends probably on the perspective of "simply": Didn't really need a computed as not reactive anyway. My brain is more wired to fetch and set data, but can do it as computed if this makes it more understandable.